oaknational/Oak-Web-Application
# Feat/pupil 798/prominent quiz stem instructions

## Description

Music year: 1979

- List of changes
Add jauntyAngleLabel component into puoil quizzes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2637--oak-web-application.netlify.thenational.academy/pupils/programmes/english-secondary-year-11-aqa/units/modern-text-first-deep-dive-169/lessons/introductions-and-conclusions-about-education-in-animal-farm/starter-quiz
2. Click on start quiz
3. You should see jaunty angle label in use
4. Click on exit quick, navigate to question 4 and see the use of jaunty angle label for multi angle question type

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
